easybuild/easyconfigs/x/x13as/x13as-1-1-b59-GCCcore-10.2.0.eb

```diff
@@ -0,0 +1,51 @@
+# Author: J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'MakeCp'
+
+name = 'x13as'
+version = '1-1-b59'
+
+homepage = 'https://www.census.gov/data/software/x13as.About_X-13.html'
+
+description = """
+X-13ARIMA-SEATS is seasonal adjustment software produced, distributed, 
+and maintained by the Census Bureau.
+
+Features of X-13ARIMA-SEATS include:
+    - Extensive time series modeling and model selection capabilities for 
+      linear regression models with ARIMA errors (regARIMA models);
+    - The capability to generate ARIMA model-based seasonal adjustment 
+      using a version of the SEATS software originally developed by Victor 
+      Gómez and Agustín Maravall at the Bank of Spain, as well as nonparametric 
+      adjustments from the X-11 procedure;
+    - Diagnostics of the quality and stability of the adjustments 
+      achieved under the options selected;
+    - The ability to efficiently process many series at once.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+source_urls = [
+    'https://www2.census.gov/software/x-13arima-seats/x13as/unix-linux/program-archives',
+]
+sources = ['%(name)s_asciisrc-v%(version)s.tar.gz']
+patches = ['%(name)s-%(version)s_makefile.patch']
+checksums = [
+    '256ee87f326ef6e0ac33a806812d614a5584c6b9e1f2a67ec06af52f2c3be85d',  # x13as_asciisrc-v1-1-b59.tar.gz
+    'cbde094a2396c256e9ac996cd539806bb0274e569d578ad42c4879f0e02c5ef3',  # x13as-v1-1_makefile.patch
+]
+
+builddependencies = [
+    ('binutils', '2.35'),
+]
+
+build_cmd = 'make -f makefile.gf'
+
+files_to_copy = [(["x13as"], "bin")]
+
+sanity_check_paths = {
+    'files': ['bin/x13as'],
+    'dirs': [],
+}
+
+moduleclass = 'tools'
```

Comment:
Should this be `math`?

Reply:
Changed. I am sometimes not really sure if that is a `tool`which is using `math` or just plain `math` being used as a `tool`
:-)
I have also updated PR #16163